redaxo/src/addons/metainfo/lib/handler/media_handler.php

```diff
@@ -187,8 +187,9 @@ public function extendForm(rex_extension_point $ep)
             unset($params['media']);
         } elseif ('MEDIA_ADDED' == $ep->getName()) {
             $sql = rex_sql::factory();
-            $qry = 'SELECT id FROM ' . rex::getTablePrefix() . 'media WHERE filename="' . $params['filename'] . '"';
-            $sql->setQuery($qry);
+
+            $qry = 'SELECT id FROM ' . rex::getTablePrefix() . 'media WHERE filename=:filename';
+            $sql->setQuery($qry, [':filename' => $params['filename']]);
```

Comment:
```suggestion
            $sql->setQuery($qry, ['filename' => $params['filename']]);
```

Wir lassen den Doppelpunkt normalerweise weg (er ist optional).